systems/notification/api-gateway/pkg/config.go

```diff
@@ -36,7 +36,7 @@ func NewConfig() *Config {
 		},
 
 		Server: rest.HttpConfig{
-			Port: 8085,
+			Port: 8089,
```

Comment:
This is the internal port for the http service inside the container, so I think  we can use `8080` here.